examples/src/bin/spi_loopback_dma_psram.rs

```diff
@@ -13,10 +13,9 @@
 //! Connect MISO and MOSI pins to see the outgoing data is read as incoming
 //! data.
 //!
-//! If your module is quad PSRAM then you need to change the `psram` feature in the
-//! in the features line below to `quad-psram`.
+//! If your module is octal PSRAM then you need to set `ESP_HAL_CONFIG_PSRAM_MODE` to `octal`.
```

Comment:
Does this mean octal PSRAM will not work if mode is quad, or does this just mean it won't run at max speed?

If it's the latter I think it should be further clarified to say so.

Reply:
won't work with the wrong setting - and unfortunately, there isn't really a way to detect the type of spiram

any suggestion regarding the wording? I thought "you need to" suggests it's not optional but maybe not

Reply:
Ah I see, the wording is fine as is then. It was just my misunderstanding of how this stuff works.